argo-workflows/sync-nb-server-to-ironic/docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 # Overview
 
-The purpose of the workflow templates in this folder is to synchronize
+The purpose of the workflow templates in this folder is to sync
```

Comment:
not sure if we need to shorten it here too

Reply:
reverted.